src/com/estafet/learning/US005/Task5.java

```diff
@@ -73,17 +73,20 @@ public static void factorial() {
 
     public static void factorialOfOddNumbers() {
         Scanner reader = new Scanner(System.in);
-        System.out.format("Please enter a positive and odd number to view Factorial of it: %n");
+        System.out.format("Please enter a positive number to view Factorial without even numbers in the sequence: %n");
         number = reader.nextInt();
 
-        if (number > 0 && number % 2 != 0) {
+        if (number > 0) {
             long i, fact = 1;
```

Comment:
why you declare local (method) vars for loops?

Reply:
changed.